src/Parser/Priest/Shared/Modules/Features/DesperatePrayer.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import ExpandableStatisticBox from 'Main/ExpandableStatisticBox';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class DesperatePrayer extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+    spellUsable: SpellUsable,
+  };
+
+  desperatePrayerUsages = [];
+  currentDesperatePrayerUsage;
```

Comment:
Please explicitly set a default value for a class property, in this case setting it to `null` would probably be best.